.pre-commit-hooks.yaml

```diff
@@ -5,5 +5,5 @@
   language: python
   language_version: python3
   pass_filenames: false
-  files: ^poetry.lock$
+  files: ^poetry\.lock$
```

Comment:
```suggestion
  files: ^(.*/)?poetry\.lock$
```

Let's keep it in sync with poetry's hooks.

Reply:
Good idea, thanks! Applied.